develop/tutorials/articles/340-data-upgrades-and-verifiers/creating-an-upgrade-process-for-your-application.markdown

```diff
@@ -42,16 +42,34 @@ First, let's specify the schema version.
 
 ### Specifying the Schema Version [](id=specifying-the-schema-version)
 
-In your module's `bnd.bnd` file, specify a `Liferay-Require-SchemaVersion`
-header with the new schema version value. Here's an example schema version
-header: 
+In your module's `bnd.bnd` file and use Service Builder, specify a 
```

Comment:
@cgoncas I'm not sure what you meant by "and use Service Builder" here.

Were you trying to mention the step that involves adding the @Reference to wait for the upgrade step execution? 

Thanks